tests/graph/test_basic.py

```diff
@@ -75,6 +87,20 @@ def perform(self, *args, **kwargs):
         raise NotImplementedError("No Python implementation available.")
 
 
+class MyCustomOp(Op):
```

Comment:
Do we need a new test Op, old ones weren't good?

Reply:
Nah they are. I was just trying something out which did not work.

Reply:
Don't forget to remove